packages/create-react-native-library/src/index.ts

```diff
@@ -365,10 +391,12 @@ async function create(argv: yargs.Arguments<any>) {
       native: languages !== 'js',
       architecture,
       turbomodule,
+      fabricView,
+      newArchitecture,
```

Comment:
Do we need `newArchitecture` ? @MateWW . `architecture` indicates, what arch is it, so IMO this is redundant